gpcc-mocks/src/main/java/uk/nhs/adaptors/gpccmocks/gpc/GpcController.java

```diff
@@ -208,6 +208,37 @@
         return new ResponseEntity<>(body, getResponseHeaders(), HttpStatus.OK);
     }
 
+    @GetMapping(value = "/fhir/Binary/{documentId}")
+    @ResponseStatus(value = ACCEPTED)
+    public ResponseEntity<String> retrieveMigrateDocument(
```

Comment:
I didn't add a separate endpoint for migrating a document, because in the spec the path is exactly the same as for the endpoint for retrieving a document.

Reply:
using docker imagine the endpoint doesn't exist for documents/fhir/Binary so I have updated this to /fhir/Binary